### ISSUE TITLE
"multipleDefine" error from Dojo when first loading Maqetta

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2955]
When first loading Maqetta, I see an error in the console:

``` javascript
{
   message: "multipleDefine",
   src: "dojoLoader",
   info: {
      mid: "maq-metadata-dojo/callbacks",
      url: "http://maqetta-app-test.austin.ibm.com/maqetta/app/metadata/dojo/1.8/callbacks.js"
   }
}
```

Are we evaluating this file twice, perhaps?

@peller @JonFerraiolo

**Comment by jhpedemonte** [comment]
When creating a new file, I see this error 2 more times.  So we're constantly trying to re-evaluate the callbacks object.

**Comment by peller** [comment]
is this related to #2690?  Where are we loading callbacks as a module, as opposed to the xhr+eval in metadata.js?

**Comment by JonFerraiolo** [comment]
-> @jhpedemonte Changing to Prio-High so that it gets attention early to make sure it truly is innocuous. It sounds a bit scary.

@peller

**Comment by jhpedemonte** [comment]
Odd, I don't see this when running from source, but I do see it on the build on maqetta-test.

**Comment by peller** [comment]
appears to be fixed with b7fac7a1e78fd8612cff598b504f83b74fc0a261
